x-pack/plugins/security/public/authentication/login/login_page.tsx

```diff
@@ -146,12 +146,15 @@ export class LoginPage extends Component<Props, State> {
     ) : (
       <EuiIcon type="logoElastic" size="xxl" />
     );
+    const logoStyle = customLogo ? { padding: 0 } : {};
```

Comment:
__nit:__ would you mind adding a clarifying comment in the code why this is needed for custom logos and not default one (for future readers)?